### ISSUE TITLE
User resource management questions and `madmin:field` problem

### ISSUE THREAD

**Issue by dwatek** [excid3/madmin#80]
Hi!

My issues are more questions about CRUD for users. I have users from devise that have password and password_confirmation. Unfortunately these are the fields that do not show up in the form for adding a new user. I understand that the only solution is to override the `form` view for `users`, and `update` action in users_controller?

Second question is about `Resources`. In a specific resource such as `UserResource < Madmin::Resource` we declare `attribute :id, form: false`. Is there somewhere a list of options that attribute can take, apart `form: false/true`?

Third thought 😄 I think `rails g madmin:field` does not work. 
```
rails g madmin:field Custom

Running via Spring preloader in process 85728
Could not find generator 'madmin:field'. Maybe you meant "madmin:view"?
Run `bin/rails generate --help` for more options.

rails g --help

Madmin:
  madmin:install
  madmin:resource
  madmin:view
  madmin:views
  madmin:views:edit
  madmin:views:form
  madmin:views:index
  madmin:views:javascript
  madmin:views:layout
  madmin:views:navigation
  madmin:views:new
  madmin:views:show
```

**Comment by excid3** [comment]
`password` and `password_confirmation` are virtual attributes. You'll just add them to the `UserResource` that's generated.

```ruby
class UserResource < Madmin::Resource
  attribute :password
  attribute :password_confirmation
end
```

I don't think the field options are documented anywhere yet. Need to do that still. You can look at the `test/dummy` app to see examples.

The field generator may not be released yet and only in `master`.

**Comment by excid3** [comment]
I made a new release yesterday with the field generator and some other improvements. 👍

**Comment by johlym** [comment]
Hey @excid3 I'm not seeing the new release neither here nor on RubyGems. 🤔

**Comment by excid3** [comment]
Whoops. 😅  

Released 1.2.1 now
